scripts/quests/jeuno/helpers.lua

```diff
@@ -499,7 +502,10 @@ function xi.jeuno.helpers.BorghertzQuests:new(params)
                         if player:getGil() >= 1000 then
                             player:delGil(1000)
                             quest:setVar(player, 'Prog', 3) -- Spoken with Yin Pocanakhu.
-                            player:updateEvent(1)
+                            player:updateEvent(1, 1, 0, 0)
+                        else
+                            player:messageText(zones[xi.zone.LOWER_JEUNO].text.YIN_POCANAKHU_GET_LOST, true)
```

Comment:
LLS is complaining about this line
```suggestion
                            player:messageText(npc, zones[xi.zone.LOWER_JEUNO].text.YIN_POCANAKHU_GET_LOST, true)
```
?